corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js

```diff
@@ -155,6 +155,11 @@ hqDefine("cloudcare/js/formplayer/menus/utils", function () {
                 }
                 hqImport('analytix/js/kissmetrix').track.event(event, eventData);
             }
+            if (/search_command.m\d/.test(menuResponse.queryKey) && menuResponse.currentPage === 0) {
```

Comment:
Do you need to escape the "." in this regular expression?

Reply:
I'm curious what this test is for - is this to ensure that the user is on a case search screen and not a casedb-based case list?

Nitpick: I'd replace `\d` with `\d+` since there can be multiple digits. It'll work either way, since there will always be at least one digit, it's just nice to be explicit about exactly what you're looking for.

Reply:
@Robert-Costello Good call, will fix since I am specifically looking for a "."

@orangejenny Yes, this is the way I was able to find that someone is using case search and not the casedb search. Maybe there's a better/cleaner way? Initially I was working with `submitAction` in views/query.js, but that missed the "Skip to Default Results" workflow.

Reply:
I wasn't sure initially, but I looked around a little bit and agree `queryKey` is the best way to go - good find.